spec/integration/query_result_builder_spec.rb

```diff
@@ -30,7 +30,7 @@ def self.id_namespace
       Object.send(:remove_const, :FooObject)
     end
     it "should return an array of objects that are of the class stored in active_fedora_model_s" do
-      query = "id\:#{RSolr.escape(@test_object.id)} OR id\:#{RSolr.escape(@foo_object.id)}"
+      query = "id\:#{ActiveFedora::SolrQueryBuilder.solr_escape(@test_object.id)} OR id\:#{ActiveFedora::SolrQueryBuilder.solr_escape(@foo_object.id)}"
```

Comment:
@jcoyne but if you do that, are we loosing the "OR" syntax? I forget which Solr defaults to, but it can be either "AND" or "OR" depending on how it's configured.